examples/blog/src/layouts/BlogPost.astro

```diff
@@ -30,22 +56,28 @@ const { title, description, pubDate, updatedDate, heroImage } = Astro.props;
 
 	<body>
 		<Header />
-		<main>
+		<main class="blog-post">
```

Comment:
Super tiny nit that's more personal preference, feel free to ignore!

I always try to lean on Astro's scoped styles when possible.  Here the style should be able to target `main { }` directly, avoiding the extra `.blog-post` class all together

Reply:
tbh not sure why I did that ha, goblin mode things